packages/dataviews/src/components/dataviews/stories/index.story.tsx

```diff
@@ -130,44 +132,68 @@ function PlanetOverview( { planets }: { planets: SpaceObject[] } ) {
 	const moons = planets.reduce( ( sum, item ) => sum + item.satellites, 0 );
 
 	return (
-		<Card isBorderless style={ { padding: '12px 24px' } }>
-			<CardHeader>
-				<Heading level={ 2 }>{ __( 'Solar System numbers' ) }</Heading>
-				<DataViews.Search label={ __( 'moons by planet' ) } />
-			</CardHeader>
-
-			<CardBody>
-				<VStack spacing={ 2 }>
-					<Text size={ 18 } as="p">
-						{ createInterpolateElement(
-							_n(
-								'<PlanetsNumber /> planet',
-								'<PlanetsNumber /> planets',
-								planets.length
-							),
-							{
-								PlanetsNumber: (
-									<strong>{ planets.length } </strong>
-								),
-							}
-						) }
-					</Text>
-
-					<Text size={ 18 } as="p">
-						{ createInterpolateElement(
-							_n(
-								'<SatellitesNumber /> moon',
-								'<SatellitesNumber /> moons',
-								moons
-							),
-							{
-								SatellitesNumber: <strong>{ moons } </strong>,
-							}
-						) }
-					</Text>
-				</VStack>
-			</CardBody>
-		</Card>
+		<>
+			<Grid
+				templateColumns="repeat(auto-fit, minmax(330px, 1fr))"
+				justify="left"
+				align="flex-start"
+				className="dataviews-sticky-header"
+			>
+				<Card variant="secondary">
+					<CardHeader>
+						<Heading level={ 2 }>
+							{ __( 'Solar System numbers' ) }
+						</Heading>
+					</CardHeader>
+
+					<CardBody>
+						<VStack>
+							<Text size={ 18 } as="p">
+								{ createInterpolateElement(
+									_n(
+										'<PlanetsNumber /> planet',
+										'<PlanetsNumber /> planets',
+										planets.length
+									),
+									{
+										PlanetsNumber: (
+											<strong>{ planets.length } </strong>
+										),
+									}
+								) }
+							</Text>
+
+							<Text size={ 18 } as="p">
+								{ createInterpolateElement(
+									_n(
+										'<SatellitesNumber /> moon',
+										'<SatellitesNumber /> moons',
+										moons
+									),
+									{
+										SatellitesNumber: (
+											<strong>{ moons } </strong>
+										),
+									}
+								) }
+							</Text>
+						</VStack>
+					</CardBody>
+				</Card>
+
+				<HStack justify="end">
+					<DataViews.Search label={ __( 'moons by planet' ) } />
+				</HStack>
+
+				<HStack justify="end">
+					<DataViews.Pagination />
+					<DataViews.ViewConfig />
```

Comment:
The grid layout doesn't display the PreviewSizePicker component:

| This PR | Default story |
| --- | --- |
| <img width="370" alt="Screenshot 2025-05-02 at 13 46 56" src="https://github.com/user-attachments/assets/b7268740-2c3e-4c62-a91d-540fc541e8f8" /> | <img width="370" alt="Screenshot 2025-05-02 at 13 47 55" src="https://github.com/user-attachments/assets/32708e42-a82d-492c-ba43-368fa4e96c9c" /> |

Reply:
fixed https://github.com/Automattic/wp-calypso/pull/103070/commits/1fedbc27c6da750d58e82a4d4aa48dcb8525a788. It was mentioned by @ntsekouras [here](https://github.com/Automattic/wp-calypso/pull/102996#discussion_r2071200052). 

I've created a follow-up WOOA7S-128